jmix-core/core/src/main/java/io/jmix/core/security/UserManager.java

```diff
@@ -29,25 +29,54 @@
 public interface UserManager {
 
     /**
-     * Changes the password for the specific user.
+     * Changes the password for the specific user. The changes will be saved to the database.
```

Comment:
Changes the password for the specific user and saves changes to the database immediately.